sr_unix/rts_error.c

```diff
@@ -69,7 +69,7 @@ int rts_error_va(void *csa, int argcnt, va_list var);
  *  =======	zero MUST be specified if there are no parameters.
  * ----------------------------------------------------------------------------------------
  */
-
+/* coverity[+kill] */
```

Comment:
?? Any idea what this is?

Reply:
Coverity (https://www.synopsys.com/software-integrity.html) is something that FIS is probably using.

Reply:
Agree. Below is what I think is the corresponding release note.

http://tinco.pair.com/bhaskar/gtm/doc/articles/GTM_V6.3-004_Release_Notes.html#GTM-8870

GT.M addresses some issues identified by static analysis tools which appear to be obscure, and, in most cases, harmless. Previously an error with a source file could leak a file descriptor, and, in odd circumstances, trigger management could fail with a segmentation violation. These issues were only observed in the GT.M development environment, and were never reported by a user. (GTM-8870)